Tests/service-test.yml

```diff
@@ -0,0 +1,65 @@
+config:
+  target: "http://localhost:4006"
+  http:
+    timeout: 10
+  phases:
+    - duration: 60
+      arrivalRate: 10
+      rampTo: 50
+    - duration: 60
+      arrivalRate: 50
+      rampTo: 75
+    - duration: 60
+      arrivalRate: 75
+      rampTo: 100
+    - duration: 300
+      arrivalRate: 100
+      rampTo: 200
+  processor: "./processor.js"
+
+scenarios:
+  - name: "View product page"
+    weight: 15
```

Comment:
DIdn't know about a weight variable for scenarios but just looked it up - pretty cool